templates/index.html

```diff
@@ -1,55 +1,189 @@
+                style="background-color: lightcoral;">Label 1</div>
+              <div class="list-group-item py-2 px-3 label" value="dolor"
+                style="background-color: yellow;">Label 2</div>
+              <div class="list-group-item py-2 px-3 label" value="rem"
+                style="background-color: pink;">Label 3</div>
+              <div class="list-group-item py-2 px-3 label" value="sit"
+                style="background-color: greenyellow;">Label 4</div>
+            </div>
+          </div>
+        </div>
+        <div class="my-3 row">
+          <div class="col-lg-3"></div>
+          <button id="export" type="button" class="btn btn-primary col-lg-6">Export</button>
+          <div class="col-lg-3"></div>
+        </div>
+        <div class="my-2 row bg-light rounded-lg">
+          <div class="col-lg-12 mt-2 mb-3">
+            <h2 class="my-2">Colors</h2>
+            <p>Please select a color</p>
+            <div class="highlight-color color-selected" style="background-color: red" id="red">
+              <p></p>
+            </div>
+            <div class="highlight-color" style="background-color: orange" id="orange">
             </div>
+            <div class="highlight-color" style="background-color: yellow" id="yellow">
+            </div>
+            <div class="highlight-color" style="background-color: green" id="green">
+            </div>
+            <div class="highlight-color" style="background-color: blue" id="blue">
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- /.col-lg-2 -->
 
+      <!-- Center Highlight Area -->
+      <div class="col-lg-7">
+        <div class="container row border rounded-lg my-2 bg-light">
+          <div id="content" class="my-4 mx-3" style="width: 100%;">
+            <textarea id="doc_view" readonly="true" style="resize: none; width: 100%; border:none;"></textarea>
+          </div>
         </div>
-        <div class="middle_panel">
-          <textarea id="doc_view" readonly="true"></textarea>
-          <br/>
-          <button id='export'>Download</button>
+      </div>
+      <!-- /.col-lg-8 -->
+
+      <!-- Annotations -->
+      <div class="col-lg-3">
+        <div class="my-2 row bg-light rounded-lg">
+          <div class="col-lg-12 mt-2 mb-3 mx-2">
+            <h2 class="my-2 ">Annotations</h2>
+            <hr>
+            </hr>
+            <div class="list-group">
+              <h4>Label 1</h4>
+              <div class="ml-3">
+                <a href="" class="annotation">word 1</a><br>
+                <a href="" class="annotation">word 2</a><br>
+              </div>
+            </div>
+            <hr>
+            </hr>
+            <div class="list-group">
+              <h4>Label 2</h4>
+              <div class="ml-3">
+                <a href="" class="annotation">word 1</a><br>
+                <a href="" class="annotation">word 2</a><br>
+                <a href="" class="annotation">word 3</a><br>
```

Comment:
These shouldn't be clickable

Reply:
I was assuming we were adding a way to click the annotation and scroll to it

Reply:
Gotcha. If that's what we're doing then this is resolved